src/marten/db/field/slug.cr

```diff
@@ -14,11 +20,27 @@ module Marten
           @null = false,
           @unique = false,
           @index = true,
-          @db_column = nil
+          @db_column = nil,
+          @slugify : Symbol? = nil,
+          @slugify_cb : (::String -> ::String) = ->(value : ::String) { generate_slug(value) }
```

Comment:
I am uncertain about the need for this field option. If a user wants to customize how slugs are generated, it might be more appropriate for them to write a `before_validation` callback and implement their own slug generation logic. In this situation, what about removing this 
`slugify_cb` option and enforcing the use of the framework's slugify capabilities instead?